libpod/container_internal_linux.go

```diff
@@ -38,6 +38,7 @@ import (
 	"github.com/containers/storage/pkg/archive"
 	"github.com/containers/storage/pkg/idtools"
 	securejoin "github.com/cyphar/filepath-securejoin"
+	"github.com/moby/sys/mountinfo"
```

Comment:
Could you use  	"github.com/containers/storage/pkg/mount"

Reply:
Ah, sure. I wasn't aware of that. I've just pushed the update. Let me know if I should squash the commits.